### PR TITLE
comment for review

### PR DESCRIPTION
**Pull request by nmduc3**

### CODE REVIEW

**Comment by nmduc3** [comment]
screenshot from 2017-12-17 18 58 10

**Comment by nmduc3** [comment]
ready

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class CommentsController < ApplicationController
+  before_action :find_review, only: :create
+  before_action :blank_comment, only: :create
+
+  def create
+    create_comment
+    create_activity @comment, "comment", current_user
```
Comment (dothidiemthao) [inline app/controllers/comments_controller.rb]: "comment" settings

File: app/views/comments/create.js.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$('#rv-comments-<%= @review.id %>').html('<%= j render @review.comments.includes :user %>');
```
Comment (dothidiemthao) [inline app/views/comments/create.js.erb]: @review.comments.includes :user cái này load ở controller em

File: app/views/reviews/_review.html.erb [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+    <li>
+      <span class="user"><%= link_to review.user.full_name, review.user %></span>
+      <p class="review-p"><%= review.content %></p>
+      <span class="timestamp">
+        <%= t "posted" %> <%= time_ago_in_words(review.created_at) %> <%= t "ago" %>.
+      </span>
+    </li>
+    <li>
+      <ul>
+        <div id="rv-comments-<%= review.id %>"><%= render review.comments.includes :user %></div>
```
Comment (dothidiemthao) [inline app/views/reviews/_review.html.erb]: review.comments.includes :user ko get data trực tiếp ở view nhé

**Comment by dothidiemthao** [comment]
commented

**Comment by nmduc3** [comment]
ready

**Pull request merged**